src/main/kotlin/no/skatteetaten/aurora/mokey/service/StorageGridObjectAreaService.kt

```diff
@@ -0,0 +1,32 @@
+package no.skatteetaten.aurora.mokey.service
+
+import no.skatteetaten.aurora.mokey.model.StorageGridObjectAreaDetails
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+
+@Service
+class StorageGridObjectAreaService(
+    val userClient: OpenShiftUserClient,
+    @Value("\${openshift.cluster}") val cluster: String,
+) {
+
+    suspend fun findAllStorageGridObjectAreasForAffiliation(affiliation: String): List<StorageGridObjectAreaDetails> {
+        return userClient.getProjectsInAffiliation(affiliation)
+            .flatMap { userClient.getStorageGridObjectAreas(it.metadata.name) }
+            .map {
+                val tenant = "$affiliation-$cluster"
+                StorageGridObjectAreaDetails(
+                    name = it.metadata.name,
+                    namespace = it.metadata.namespace,
+                    creationTimestamp = it.metadata.creationTimestamp,
+                    bucketPostfix = it.spec.bucketPostfix,
```

Comment:
jeg tenker vi ikke trenger bucketpostfix. Hva tenker du @stromland ?

Reply:
Ja, den kan nok bort. Så får vi evt ta den inn igjen hvis vi ser behov for den.